### ISSUE TITLE
Buttons on the map viewer are still clickable when the map is still loading 

### ISSUE THREAD

**Issue by nanjiangshu** [MetabolicAtlas/MetabolicAtlas#1109]
**Describe the bug**

When the map is loading, prev/next buttons on the search-bar are still clickable and changing to the next hit on click. 

**Steps to reproduce**

Reproduce by setting a delay during map loading

**Expected behavior**

- [ ] Buttons that will change the visualization of map should be disabled when the map is still loading 

**Screenshots**

**Additional context**

Unsure if this is applicable to both 2D and 3D?
A screen recording would be very useful to understand the issue?

**Comment by nanjiangshu** [comment]
Hi @MalinAhlberg , @mihai-sysbio , @inghylt and @e0, I tried  to limit the network speed to 256k/s with `wondershaper` and then search in the map viewer, but it seems I could not reproduce the buggy behavior. However, when I tried with an older version in March, I was able to reproduce it as shown in the attached video. I guess the bug has been fixed in the recent changes, most likely the upgrade of vue3. So I think this issue can be closed. 
https://images.zenhubusercontent.com/53664497/f09b5506-9481-43d8-9e5c-031f1cb64e7f/map_viewer_button_clickable_error.mov
